node/task.ts

```diff
@@ -457,7 +457,7 @@ export function getDelimitedInput(name: string, delim: string, required?: boolea
 export function filePathSupplied(name: string): boolean {
     // normalize paths
     var pathValue = this.resolve(this.getPathInput(name) || '');
-    var repoRoot = this.resolve(this.getVariable('build.sourcesDirectory') || '');
+    var repoRoot = this.resolve(this.getVariable('build.sourcesDirectory') || this.getVariable('System.DefaultWorkingDirectory') || '');
```

Comment:
Just for code consistency, make this system.defaultWorkingDirectory (casing).  It ultimately doesn't matter since it's all upper cased but looks more consistent.  Also break it on another line after the || and align it.